docs/content/how-to/hibernate-workers.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Hibernating Hosted Control Plane Worker nodes 
+---
+
+# Hibernating Hosted Control Plane Worker nodes 
+
+## Context and Considerations
+
+The main reason to go through this scenario it's mainly to save resources and money, once you are not using an already created Hosted Control Plane.
+
+In order to continue with the next steps, we need to have in mind some considerations:
+
+- **This is a destructive action**, all the workloads in the worker nodes will dissapear.
+- The Hosted Control Plane will stay up and running, and you can scale up the node pool whenever you want.
+- Some pods in the control plane will stay in "Pending" state.  
+- Once you rescale the NodePool/s it will take time until they reach the fully **Ready** state.
+- We will add an annotate to the nodes which will ensure the pod drainning does not happen. This we will save time and money and also we will avoid stucked pods.
+
+Now let's explain the workflow.
+
+## Workflow
```

Comment:
Let's elaborate this section presenting our semantics and limitations. E.g:
"To temporarily remove you data plane compute all you need to do is scaling all NodePools down to zero. By default draining will be honoured to protect any workloads. You can choose to deliberately skip draining by setting [drainTimeout](https://github.com/openshift/hypershift/blob/main/api/v1alpha1/nodepool_types.go#L130-L137) to a lower value - Caveat: changing drainTimeout in an existing NodePool will trigger a rolling update first and so the new value will only affect to the new created Machines. If you are in a situation where the intent is skip draining and don't want to assume the rolling upgrade triggered by changing the field, you can skip draining forcefully in a given Machine by setting "machine.cluster.x-k8s.io/exclude-node-draining=". This is a known limitation and we plane to prevent changing drainTimeout from triggering a rolling update in the future. Machines.machine.cluster.x-k8s.io are considered a lower level resource and any consumer interaction is recommended to happen via NodePool.